### ISSUE TITLE
Could you share the config details of evaluation ?

### ISSUE THREAD

**Issue by ycszen** [PingoLH/FCHarDNet#19]
Hi, thanks for sharing the nice work.

Could you share the config details of evaluation?  I find there are some arguments requiring to set.

To achieve the 76.0% mean IoU on the Cityscapes, did you use the filpped input?  
Meanwhile, how to update bn?

Thanks again. Looking forward to your reply.

**Comment by PingoLH** [comment]
Hi, please just use hardnet.yml for both training and evaluation, where all parameters are aligned with our experiments. We did not use flipped input during evaluation. If you want to update the running mean and var for bn, please add --update_bn when running validate.py. You can choose to update bn on training set or test set by setting the yml file. However, this is not recommended since it did not improve the accuracy in our experiments.
